src/driver/kvm.rs

```diff
@@ -20,6 +20,41 @@ pub struct Kvm<T: KVMIntrospectable> {
     vec_events: Vec<Option<KVMiEvent>>,
 }
 
+impl From<kvm_segment> for SegmentReg {
```

Comment:
I'd prefer it if you didn't separate the kvm struct from its impl block here.

Reply:
Can you please elaborate. I don't quite get it.